lib/SILGen/SILGenExpr.cpp

```diff
@@ -5993,9 +5993,9 @@ RValue RValueEmitter::visitMoveExpr(MoveExpr *E, SGFContext C) {
 
 RValue RValueEmitter::visitMacroExpansionExpr(MacroExpansionExpr *E,
                                               SGFContext C) {
-  auto *rewritten = E->getRewritten();
-  assert(rewritten && "Macro should have been rewritten by SILGen");
-  return visit(rewritten, C);
+  if (auto *rewritten = E->getRewritten())
+    return visit(rewritten, C);
+  return RValue();
```

Comment:
Can this fail now?

Reply:
Reverted this portion as I'm no longer handling BraceStmt top-level macros as Expr.